uxarray/grid/grid.py

```diff
@@ -14,7 +14,7 @@
 from uxarray.io.utils import _parse_grid_type
 from uxarray.grid.area import get_all_face_area_from_coords
 from uxarray.grid.coordinates import (
-    _populate_centroid_coord,
+    _populate_face_centroids,
```

Comment:
Looks like you still need to import `_populate_edge_centroids` and add the population to each of our properties.